[
  {ex1_throw_exception, []}
].
